recipes/BEHST/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+mkdir $PREFIX/../results/
+
+mkdir $PREFIX/../temp/
+
+
+sh project.sh $PREFIX/../data/pressto_LUNG_enhancers.bed DEFAULT_EQ DEFAULT_ET
```

Comment:
`$PREFIX` is the install path, I'm not actually sure where `$PREFIX/..` ends up being when conda installs the package. It's possible it would work on travis-ci, but I think would be broken when someone installs it into their environment.

To avoid cluttering with extra nonstandard dirs `/results` and `/temp`, these should go in `/share`.  Specifically `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM`. See https://bioconda.github.io/guidelines.html#fsh-section and, probably more useful in this case, https://bioconda.github.io/guidelines.html#java.